README.md

```diff
@@ -477,6 +481,27 @@ Default: `3`
 
 Default: `True`
 
+<dt>notifier
```

Comment:
`notifier` and `version` are not intended to be changed by the user, so I don't think it makes sense to have them in the docs.